net/crowdsec/Makefile

```diff
@@ -0,0 +1,160 @@
+#
+# Copyright (C) 2021 Gerald Kerma
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=crowdsec
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/crowdsecurity/crowdsec
+PKG_SOURCE_VERSION:=73e0bbaf93070f4a640eb5a22212b5dcf26699de
+PKG_SOURCE_DATE:=20210707
+PKG_MIRROR_HASH:=940e6e122097468ad99fcee2db569afda94fc2a65837c74b0ee71bbb5c64518c
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/crowdsecurity/crowdsec
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/crowdsec/Default
+  TITLE:=CrowdSec is a free, modern & collaborative behavior detection engine
+  URL:=https://github.com/crowdsecurity/crowdsec/
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+  HOST_DEPENDS:=jq
```

Comment:
I have never seen this. What is this?

Reply:
May be it is HOST_BUILD_DEPENDS ?

Reply:
PKG_BUILD_DEPENDS ?

Reply:
> I have never seen this. What is this?

I need jq on the host for building the CrowdSec package...
How can I tell this in the Makefile ?

Reply:
That should be something like HOST_BUILD_DEPENDS:=jq/host

Is there even a jq host package here?

Reply:
> Is there even a jq host package here?

No ! There is only an utils/jq but not for host.

Reply:
I have add a jq modified package for host integration...

Reply:
It is PKG_BUILD_DEPENDS:=jq/host (with my modified jq Makefile)